migrations/20220219215644_create_books_favorites.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema.createTable('books', (table) => {
```

Comment:
Nice way to combine the creation of both sets of data tables in the same function